core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java

```diff
@@ -150,17 +152,13 @@ private Block createIntermediateValues(GroupedAggregator aggregator, GroupByIdBl
             return builder.build();
         }
 
-        private Page createValues(TestingFunctionResolution functionResolution, DecimalType type, ValueWriter writer)
+        private Page createValues(TestingFunctionResolution functionResolution, Type type)
         {
             TestingAggregationFunction implementation = functionResolution.getAggregateFunction(QualifiedName.of(function), fromTypes(type));
             partialAggregatorFactory = implementation.createAggregatorFactory(PARTIAL, ImmutableList.of(0), OptionalInt.empty());
             finalAggregatorFactory = implementation.createAggregatorFactory(FINAL, ImmutableList.of(0), OptionalInt.empty());
 
-            BlockBuilder builder = type.createBlockBuilder(null, ELEMENT_COUNT);
-            for (int i = 0; i < ELEMENT_COUNT; i++) {
-                writer.write(builder, i);
-            }
-            return new Page(builder.build());
+            return new Page(createRandomBlockForType(type, ELEMENT_COUNT, 0.03f));
```

Comment:
does it make sense to have `nullRate` benchmark param to support both nullable and non nullable cases?

Reply:
added nullRate param